erpnext/selling/report/sales_order_analysis/sales_order_analysis.py

```diff
@@ -259,6 +264,12 @@ def get_columns(filters):
 			"fieldname": "delay",
 			"fieldtype": "Data",
 			"width": 100
+		},
+		{
+			"label": _("Time Taken to Deliver"),
+			"fieldname": "time_taken_to_deliver",
+			"fieldtype": "Data",
```

Comment:
Can we make this a Duration field? And store the seconds difference between `posting_date` & `transaction_date`?

Reply:
Changed to Duration